pkg/controller/authentication/ingress.go

```diff
@@ -758,6 +759,12 @@ func platformOidcBlockIngress(instance *operatorv1alpha1.Authentication, scheme
 func platformOidcIngress(instance *operatorv1alpha1.Authentication, scheme *runtime.Scheme) *net.Ingress {
 
 	reqLogger := log.WithValues("Instance.Namespace", instance.Namespace, "Instance.Name", instance.Name)
+	var xframeDomain string
+	if (instance.Spec.Config.XFrameDomain != "") {
+		xframeDomain = strings.Join([]string{"'ALLOW-FROM ", instance.Spec.Config.XFrameDomain, "'"}, "")		
```

Comment:
you can set a flag here here called ingressUpdateRequired and Update the corresponding ingress as we upate the configmap here https://github.com/IBM/ibm-iam-operator/blob/master/pkg/controller/authentication/configmap.go#L173-L178